test/src/test/java/org/corfudb/infrastructure/LayoutServerTest.java

```diff
@@ -190,15 +197,15 @@ public void checkLayoutPersisted() throws Exception {
         final long NEW_EPOCH = 100;
 
         newLayout.setEpoch(NEW_EPOCH);
-        setEpoch(NEW_EPOCH);
+        setEpoch(NEW_EPOCH).join();
 
         // Start the process of electing a new layout. But that layout will not take effect
         // till it is committed.
-        sendPrepare(NEW_EPOCH, 1);
-        Assertions.assertThat(getLastMessage().getMsgType()).isEqualTo(CorfuMsgType.LAYOUT_PREPARE_ACK);
+        LayoutPrepareResponse resp = sendPrepare(NEW_EPOCH, 1).join();
+        Assertions.assertThat(resp.getLayout()).isNull();
+        Assertions.assertThat(resp.getRank()).isEqualTo(-1);
```

Comment:
The -1 should be a named constant. It has a significance. This should be also done in the actual code.